docs/components/datetimepicker.md

```diff
@@ -0,0 +1,409 @@
+    {
+      section: "Last",
+      values: [
+        { key: "15m", prefix: "Last", timeframeText: "15 minutes", start: function() {}, end: function() {} },
+        { key: "12h", prefix: "Last", timeframeText: "12 hours", start: function() {}, end: function() {} },
+        { key: "24h", prefix: "Last", timeframeText: "24 hours", start: function() {}, end: function() {}},
+        ...
+      ]
+    },
+    {
+      section: "Current",
+      values: [
+        { key: "current_week", prefix: "Current", timeframeText: "week", start: function() {}, end: function() {} },
+        { key: "current_month", prefix: "Current", timeframeText: "month", start: function() {}, end: function() {} }
+      ]
+    }
+    {
+      section: "Previous",
+      values: [
+        { key: "previous_week", prefix: "Previous", timeframeText: "week", start: function() {}, end: function() {} },
+        { key: "previous_month", prefix: "Previous", timeframeText: "month", start: function() {}, end: function() {} }
+      ]
+    }
+  ]
+/>
+```
+
+### Relative time frames only
+
+Set the `v-model` to [Range date time picker](#range-date-time-picker-v-model)
+Same time frames as the previous example, except now we're passing in a pre-selected time frame.
+
+<div>
+  <KDateTimePicker
+    v-model="currentValue6"
+    @change="newVal => emitVal6 = newVal"
+    placeholder="Please select a time frame"
+    mode="relative"
+    :timePeriods="exampleTimeFrames"
+  />
+  <div class="mt-6">Emitted value: <pre class="json">{{ emitVal6 }}</pre></div>
+</div>
+
+```html
+<KDateTimePicker
+  v-model="currentValue"
+  @change="newVal => emitVal = newVal"
+  placeholder="Please select a time frame"
+  mode="relative"
+  :timePeriods=[
+    {
+      section: "Last",
+      values: [
+        { key: "15m", prefix: "Last", timeframeText: "15 minutes", start: function() {}, end: function() {} },
+        { key: "12h", prefix: "Last", timeframeText: "12 hours", start: function() {}, end: function() {} },
+        { key: "24h", prefix: "Last", timeframeText: "24 hours", start: function() {}, end: function() {} },
+        ...
+      ]
+    },
+    {
+      section: "Current",
+      values: [
+        { key: "current_week", prefix: "Current", timeframeText: "week", start: function() {}, end: function() {} },
+        { key: "current_month", prefix: "Current", timeframeText: "month", start: function() {}, end: function() {} }
+      ]
+    }
+    {
+      section: "Previous",
+      values: [
+        { key: "previous_week", prefix: "Previous", timeframeText: "week", start: function() {}, end: function() {} },
+        { key: "previous_month", prefix: "Previous", timeframeText: "month", start: function() {}, end: function() {} }
+      ]
+    }
+  ]
+/>
+```
+
+## Props
+
+### v-model
+
+#### Single date time picker `v-model`
+
+A single date or time picker instance will emit a Javascript Date object as a `string`.
+
+```js
+"2022-09-06T19:45:54.406Z"  // ISO 8601 Date string
+```
```

Comment:
This talks about the values that are emitted, but can you instead show an example of how a user would set the `v-model` value?

nit: `JavaScript` not `Javascript`
```suggestion
A single date or time picker instance will emit a JavaScript Date object as a `string`.

```js
"2022-09-06T19:45:54.406Z"  // ISO 8601 Date string
```
```

Reply:
Something like this?

> A single date or time picker instance which can be seeded with a value like so:
> ```
>  <KDateTimePicker
>     v-model="currentValue"
>     ...
>  />
> ```
> Where `currentValue` is a valid Date object (eg: set to `new Date()`).

Reply:
Yep. I think the emits details you have here (for both) can be moved down to the **Events** section (you don't have to use that table format down there if the layout is weird)

Reply:
Moved all emitted notes to the ` ## Events` section. Let me know if that's clear enough.